frontend/src/app/buyer/fundraiser/[id]/item/[itemId]/page.tsx

```diff
@@ -0,0 +1,182 @@
+      {/* Back button - Mobile only */}
+      <Link
+        href={`/buyer/fundraiser/${fundraiserId}`}
+        className="md:hidden fixed top-5 left-5 z-50 rounded-full transition-colors flex-shrink-0 flex items-center justify-center p-1"
+        style={{ backgroundColor: "rgba(178, 178, 178, 0.21)" }}
+      >
+        <ChevronLeft strokeWidth={2} className="h-8 w-8 text-stone-800" />
+      </Link>
+
+      {/* Image */}
+      <div className="relative w-full h-[353px] overflow-hidden flex-shrink-0">
+        {item.imageUrl ? (
+          <Image
+            src={item.imageUrl}
+            alt={item.name}
+            fill
+            className="object-cover"
+            style={{ objectFit: 'cover' }}
+          />
+        ) : (
+          <div className="w-full h-full bg-gray-200" />
+        )}
+      </div>
+
+      {/* Content */}
+      <div className="flex flex-col px-5 pt-[22px] gap-[22px] pb-0 md:pb-20 flex-shrink-0">
+        {/* Title and Price */}
+        <div className="flex flex-col gap-[4px]">
+          <h1 className="text-[22px] font-semibold leading-[33px] text-black">
+            {item.name}
+          </h1>
+          <p className="text-base font-normal leading-[24px] text-black">
+            ${Number(item.price).toFixed(2)}
+          </p>
+        </div>
+
+        {/* Divider */}
+        <div className="h-px w-full bg-[#f6f6f6]" />
+
+        {/* Description */}
+        <div className="flex flex-col gap-2">
+          <p className="text-base font-normal leading-[24px] text-black">
+            {item.description}
+          </p>
+        </div>
+
+        {/* Quantity Selector */}
+        <div className="flex flex-col gap-3">
+          <p className="text-sm font-semibold leading-[21px] text-black">
+            Quantity
+          </p>
+          <div className="border border-[#dddddd] rounded-md w-fit">
+            <div className="flex items-center gap-[10px] p-2">
+              <button
+                onClick={handleDecrement}
+                className="p-0.5 rounded-lg hover:bg-gray-100 transition-colors"
+                disabled={quantity <= 1}
+              >
+                <Minus className="h-[18px] w-[18px] text-black" />
+              </button>
+              <p className="text-base font-semibold text-[#545454] min-w-[7px] text-center">
+                {quantity}
+              </p>
+              <button
+                onClick={handleIncrement}
+                className="p-0.5 rounded-lg hover:bg-gray-100 transition-colors"
+              >
+                <Plus className="h-[18px] w-[18px] text-black" />
+              </button>
+            </div>
+          </div>
+        </div>
+
+        {/* Add to Cart Button */}
+        <button
+          onClick={handleAddToCart}
+          className="bg-black text-white rounded-lg h-[50px] flex items-center justify-center gap-2 px-12 py-3"
```

Comment:
The buttons here and above should be shadcn <Button/>, not the default html button